Admin/AbstractAdmin.php

```diff
@@ -1042,7 +1042,7 @@ public function getActiveSubclassCode()
     /**
      * {@inheritdoc}
      */
-    public function getBatchActions()
+    final public function getBatchActions()
```

Comment:
This is a huge BC break because lot of people are overriding this method.

I would like a BC way before on `3.x` we can try to detect if the method is called form a child and then trigger a deprecated error.

Apparently, you can do it this way (example 2 seems proper): http://stackoverflow.com/a/25292439/1731473

Would you like to try it?